### PR TITLE
chore: bump version to 6.0.22

### PR DESCRIPTION
**Pull request by Decodetalkers**
Log: as title

### CODE REVIEW

**Review by BLumia — DISMISSED** [review]

**Comment by Decodetalkers** [comment]
* https://github.com/linuxdeepin/developer-center/issues/4469

记录下，等daemon集成进去了再提集成

**Comment by Decodetalkers** [comment]
集成需要跟着dtk, daemon, dde-appearance一起集成一起测试，备注

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+  * fix the case that on archlinux, it shows uniontech in the module of systeminfo
+  * fix the issue that dde-control-center coredump on opensuse
+  * fix wrong usage of the dbus mouse speed
+  * avoid hardcode of dbus service
+  * dark png shown in personalization
+  * fix dcc-update-plugin cannot initialize very fast
+  * change account to user
+  * region set in datetime plugin
+
+ -- chenhongtao <chenhongtao@deepin.org>  Thu, 08 Aug 2023 13:55:03 +0800
```
Comment (BLumia) [inline debian/changelog:14]: 8/8 是星期二

**Review by BLumia — APPROVED** [review]

**Pull request merged**